bin/ed

```diff
@@ -235,7 +235,13 @@ sub input {
     # sanity check addresses
     foreach my $ad (@adrs) {
         next unless defined $ad;
-        if ($ad > maxline() || $ad < 0) {
+        if ($ad == -1) {
```

Comment:
Maybe some constant names for these magic numbers?

Reply:
Good idea, added.